.circleci/config.yml

```diff
@@ -170,7 +170,7 @@ jobs:
       - store_artifacts:
           path: ./src/api/log/
           destination: minitest
-  backend:
+  backend_test:
```

Comment:
Why do you rename this?

Reply:
See the commit log - to free the namespace

Reply:
(the story is that we need the container to be named 'backend' so that the cassettes will work)

Reply:
If we are going down that route you mean... :-)

Reply:
well, let me put it this way: we call the other jobs minitest and rspec, not api and webui.

Reply:
I don't care. This is unrelated. Drop it please

Reply:
Na gut, weil Du es bist